swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -92,33 +92,6 @@ final class IterableAPIInternal: NSObject, PushTrackerProtocol, AuthProvider {
         self.dependencyContainer.createInAppManager(config: self.config, apiClient: self.apiClient, deviceMetadata: deviceMetadata)
```

Comment:
I am not sure I like moving all functionality into an extension. 

The `deprecated` extension is fine.

Reply:
Same response as here: https://github.com/Iterable/swift-sdk/pull/329#discussion_r445080195